protocol/chat.go

```diff
@@ -111,6 +112,16 @@ func (c *Chat) MembersAsPublicKeys() ([]*ecdsa.PublicKey, error) {
 	return stringSliceToPublicKeys(publicKeys, true)
 }
 
+func (c *Chat) JoinedMembersAsPublicKeys() ([]*ecdsa.PublicKey, error) {
+	var publicKeys []string
+	for _, item := range c.Members {
+		if item.Joined {
+			publicKeys = append(publicKeys, item.ID)
+		}
+	}
```

Comment:
No problem with the logic of this, but the human readability is more intuitive if the `item` is named after what it represents.

Then you get a nice line `if member.Joined`. At least that's my preference.

```suggestion
	for _, member := range c.Members {
		if member.Joined {
			publicKeys = append(publicKeys, member.ID)
		}
	}
```